netstandard/ref/System.Xml.Serialization.cs

```diff
@@ -56,6 +57,10 @@ public partial interface IXmlTextParser
         bool Normalized { get; set; }
         System.Xml.WhitespaceHandling WhitespaceHandling { get; set; }
     }
+    public abstract partial class SchemaImporter
```

Comment:
What's the purpose of this Type?

Reply:
IIRC It is an empty base type that we included because it shows up in various places but we don't fully implement because it has dependencies on CodeDom which isn't part of the standard.

Reply:
Correct